index.html

```diff
@@ -0,0 +1,140 @@
+                                                <img src="./assets/images/workshop.png" alt="">
+                                                <h3 class="box-heading">MD agencies</h3>
+                                                <p class="bos-paragraph">connect with the worlds best modeling agencies and be among the experts</p>
+                                                        </div>
+                                                        <div class="box">
+                                                            <img src="./assets/images/ignite.png" alt="">
+                                                            <h3 class="box-heading">lecture</h3>
+                                                            <p class="bos-paragraph">Lorem ipsum dolor sit amet consectetur adipisicing elit. Vel quia nam animi </p>
+                                                                    </div>
+    </div>
+            <button class="join-button" type="button">join cc global summit</button>                                           
+    </section>
+    <!-- <div id="featureActs" class="featured-acts">
+        <span class="featured-title">Featured Acts</span>
+        <hr class="featured-solid" />
+        <div id="actContainer" class="acts-container"></div>
+        <button type="button" id="collapse" class="btn btn-collapse">
+          <span id="expand">More<i class="fas fa-angle-down"></i></span>
+          <span id="exit" class="hide">Less<i class="fas fa-angle-up"></i></span>
+        </button>
+      </div> -->
+      <div class="about-main-heding">
+        <h2 class="p-head">participants and MC's</h2>
+        <hr>
+      </div>
+      
+      <section id="about" class="about">
+     <h6>h</h6>
+            </section>
+            <div class="button">
+              <button class="see-button" id="see-button" type="button">see more <i class="fa-solid fa-chevron-down"></i></button>
+              
+          </div>
+           
+            
+            <section class="paterners">
+              <div>
+                <h5>paterners</h5>
+                <hr>
+              </div>
+              <div class="patteners-imglogo">
+                <img  class="patterners-logo" src="./assets/images/airbnb.png" alt="" >
+                <img class="patterners-logo"  src="./assets/images/daum.png" alt="" >
+                <img class="patterners-logo"  src="./assets/images/naver.png" alt="" >
+                <img class="patterners-logo" src="./assets/images/.png" alt="" >
+                <img class="patterners-logo" src="./assets/images/.png" alt="" >
+              </div>
+             
+            </section>
+            <footer>
+              <div>
+              <img class="logo-image" src="./assets/images/logo1.png" alt="">
+              </div>
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet, quae voluptatum libero enim ab 
+               .</p>
+            </footer>
```

Comment:
- [x] Your Desktop Size should be divided into a top `dark bar` and a bottom `white bar`. Kindly change the background color and the text color of the Home Page footer, you have to stick to the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015),

- Your design:

![foot11](https://user-images.githubusercontent.com/79058364/169177455-4dd88b0b-0d9f-4f6f-96c9-5e73fc2bab7e.png)


- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![foot1](https://user-images.githubusercontent.com/79058364/168645688-6f1736a4-f127-4f6f-8eca-363f70ad7904.png)